saleor/static/dashboard-next/types.ts

```diff
@@ -9,15 +9,37 @@ export interface UserError {
   message: string;
 }
 
+export interface ListSettings {
+  name: string;
+  rowNumber: number;
+}
+
+export enum Lists {
+  CATEGORY_LIST = "CATEGORY_LIST",
+  COLLECTION_LIST = "COLLECTION_LIST",
+  CUSTOMER_LIST = "CUSTOMER_LIST",
+  DRAFT_LIST = "DRAFT_LIST",
+  NAVIGATION_LIST = "NAVIGATION_LIST",
+  ORDER_LIST = "ORDER_LIST",
+  PAGES_LIST = "PAGES_LIST",
+  PRODUCT_LIST = "PRODUCT_LIST",
+  SALES_LIST = "SALES_LIST",
+  SHIPPING_METHODS_LIST = "SHIPPING_METHODS_LIST",
+  STAFF_MEMBERS_LIST = "STAFF_MEMBERS_LIST",
+  VOUCHER_LIST = "VOUCHER_LIST"
+}
+
 export interface ListProps {
   disabled: boolean;
   pageInfo?: {
     hasNextPage: boolean;
     hasPreviousPage: boolean;
   };
+  listSettings?: ListSettings;
```

Comment:
This interface is named `ListProps` and I'm not sure if naming prop `listSettings` is any more descriptive than just `settings`